.circleci/config.yml

```diff
@@ -441,33 +441,6 @@ jobs:
       docker_layer_caching: true
     steps:
       - checkout
-      - docker_login:
-          username: $DOCKER_USER
-          password: $DOCKER_PASS
-      - run:
-          name: Deploy to Dockerhub
-          command: |
-            ./scripts/store_git_info.sh
-            make build_prod
```

Comment:
We'll need to keep these two lines since the GAR push below starts at tagging but doesn't include the build step